megazords/ios/MozillaAppServicesTests/PlacesTests.swift

```diff
@@ -57,62 +57,54 @@ enum CheckChildren {
 }
 
 // similar assert_json_tree from our rust code.
-func checkTree(_ n: BookmarkItem, _ want: [String: Any], checkChildren: CheckChildren = .full) {
```

Comment:
These test changes may look a little scary, but really I'm just reverting the changes I made in my bookmarks PR. See [the this file in main](https://github.com/mozilla/application-services/blob/main/megazords/ios/MozillaAppServicesTests/PlacesTests.swift) for details.